activestorage/lib/active_storage/attached/many.rb

```diff
@@ -49,7 +49,8 @@ def blobs
     def attach(*attachables)
       if record.persisted? && !record.changed?
         record.public_send("#{name}=", blobs + attachables.flatten)
-        record.save
+        return record.public_send("#{name}") if record.save
+        false
```

Comment:
```suggestion
        if record.save
          record.public_send("#{name}")
        else
          false
```

Reply:
Thank you for the review, I've made the suggested changes